src/sub/tests/conftest.py

```diff
@@ -56,46 +61,71 @@ def get_file(filename, path=THIS_PATH, **overrides):
 
 def pytest_configure():
     """Called before testing begins"""
-    global ddb_process
     for name in ("boto3", "botocore"):
         logging.getLogger(name).setLevel(logging.CRITICAL)
-    if os.getenv("AWS_LOCAL_DYNAMODB") is None:
-        os.environ["AWS_LOCAL_DYNAMODB"] = f"http://127.0.0.1:{CFG.DYNALITE_PORT}"
 
-    # Latest boto3 now wants fake credentials around, so here we are.
     os.environ["AWS_ACCESS_KEY_ID"] = "fake"
     os.environ["AWS_SECRET_ACCESS_KEY"] = "fake"
```

Comment:
not part of your code change, but i dont know that we need AWS_ACCESS_KEY_ID or AWS_SECRET_ACCESS_KEY anymore.  Those values are set in dodo.py and available as CFG values throughout our code.

Reply:
Agreed.  I am going to leave that alone though on this pull request (PR) as it is functionally unrelated to the work.